src/app/sb-admin/components/pages/sb-organization/sb-organization.component.ts

```diff
@@ -39,40 +45,185 @@ export class SbOrganizationComponent implements OnDestroy {
     }
   };
 
+  constructor(private orgList: OrganizationListService, private userService: UserService,
+    private userCountService: UserCountService, public dialogService: DialogService,
+    public ref: DynamicDialogRef, private messageService: MessageService, private i18nextPipe: I18NextPipe) { }
 
-  constructor(private orgList: OrganizationListService, public dialogService: DialogService, public ref: DynamicDialogRef, private messageService: MessageService, private i18nextPipe: I18NextPipe) { }
   ngOnInit() {
-    this.getAllOrganizationList();
+    this.getTotalOrgCount();
+    this.getTotalUserCount();
+    this.getTotalSubOrgCount();
+  }
+
+  loadOrganizationData(event: any) {
+    this.loading = true;
+    this.getAllOrg(event).subscribe((data: any) => {
+      this.organizationDetail = data;
+      if (data && data.length > 0) {
+        this.getSubOrgCountOfEachOrg(data);
+        this.getUserCountOfEachOrg(data);
+      } else {
+        this.loading = false;
+      }
+    });
   }
 
-  getAllOrganizationList() {
+  getAllOrg(event: any) {
+    let offset = (event.first) / 10 * this.pageOffsetConstant;
```

Comment:
10 should it be hardcoded or shall come from rowsPerPageOptions / count as selected by user?

Reply:
sir it's  hard coded i am setting this 10 in variable and basically used for get offset value as per page